components/availability/src/Availability.svelte

```diff
@@ -269,7 +270,12 @@
   $: (async () => {
     if (_this.booking_options) {
       consecutiveOptions = _this.booking_options;
-    } else if (id && Array.isArray(_this.events) && dayRange.length > 0) {
+    } else if (
+      !loading &&
+      id &&
+      Array.isArray(_this.events) &&
```

Comment:
Is this check needed since events is typed as `EventDefinition[]`?

Reply:
Yeah, since the typing doesn't actually enforce anything. It's just making sure that the value isn't falsy and that the user didn't pass anything weird in.